packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -54,6 +58,9 @@ contract Exchange is
   uint256 public updateFrequency;
   uint256 public minimumReports;
 
+  uint256 public minSupplyForStableBucketCap;
+  FixidityLib.Fraction public maxStableBucketFraction;
```

Comment:
I'm the worst at naming variables, but I don't know if this feels very descriptive either

Reply:
You should probably also add this to the initialization function